### PR TITLE
#update help

### PR DESCRIPTION
**Pull request by FedericoCarli**

### CODE REVIEW

File: R/1b-base-model-function.R [hunk 1]
```diff
@@ -19,14 +19,16 @@
 #'                      order of the event tree.
 #'          \item stages: A named list where each component
 #'                        stores the stages for the given variable.
+#'          \item ctables: The contingency tables of the data.
+#'          \item lambda: The smoothing parameter used to estimate the model.
 #'          \item prob: The conditional probability tables for every
 #'                      variable and every stage.
-#'          \item ctables: The contingency tables of the data.
-#'          }
+#'          \item ll: The estimated log-likelihood associated to the model.
```
Comment (gherardovarando) [inline R/1b-base-model-function.R]: The log-likelihood of the `estimated` model.

File: R/1b-base-model-function.R [hunk 2]
```diff
@@ -375,14 +384,32 @@ join_stages <- function(object, v,  s1, s2) {
 #' Randomly assign some of the path to a new stage
 #'
 #' @param object a staged event tree object
-#' @param var the variable where to split the stage
-#' @param stage the name of the stage
-#' @param p probability
+#' @param var the variable where to split the original stage into two stages
```
Comment (gherardovarando) [inline R/1b-base-model-function.R]: revert to previous:
the variable where to split the stage

File: R/1b-base-model-function.R [hunk 3]
```diff
@@ -375,14 +384,32 @@ join_stages <- function(object, v,  s1, s2) {
...
+#' @param stage the name of the original stage
```
Comment (gherardovarando) [inline R/1b-base-model-function.R]: the name of the stage to split

File: R/1b-base-model-function.R [hunk 4]
```diff
@@ -486,10 +513,12 @@ indep.data.frame <- function(x, fit = TRUE, lambda = 0, ...) {
 #' @param object1 first staged event tree to compare
 #' @param object2 second staged event tree to compare
 #'
-#' @return list with inclusion relations between stage structures of each variable of the dataset
+#' @return a list with inclusion relations between stage structures of each variable of the dataset
```
Comment (gherardovarando) [inline R/1b-base-model-function.R]: @return a list with ... of each variable in the model.

File: R/1b-base-model-function.R [hunk 5]
```diff
@@ -47,8 +49,7 @@ staged_ev_tree.default <- function(x, ...) {
 #' @examples
 #'
 #' ######### from table
-#' data("Titanic")
-#' modT <- staged_ev_tree(Titanic, fit = TRUE, full = TRUE, lambda = 1)
+#' modT <- staged_ev_tree(PhDArticles, fit = TRUE, full = TRUE, lambda = 1)
```
Comment (gherardovarando) [inline R/1b-base-model-function.R]: Here the example should be to create a staged event tree from a contingency table, that is why I was using Titanic data.

File: R/1b-base-model-function.R [hunk 6]
```diff
@@ -328,6 +329,14 @@ set_stage <- function(sevt, path, stage) {
 #' @param s1 first stage
 #' @param s2 second stage
 #' @return the staged event tree where \code{s1} and \code{s2} are joined
+#' @details this function joins together two stages associated to the same variable.
+#' It can be used to manually force two situations to be in the same stage.
```
Comment (gherardovarando) [inline R/1b-base-model-function.R]: remove: it can be used ....

**Comment by FedericoCarli** [comment]
changed all the comments you did

**Pull request merged**